docs/guides/setup/configuring-metadata-stores/how-to-configure-an-expectation-store-to-postgresql.md

```diff
@@ -1,5 +1,132 @@
+            store_backend:
+                class_name: DatabaseStoreBackend
+                credentials: ${db_creds}
+    ```
+
+
+4. **Confirm that the new Expectations store has been added by running** ``great_expectations --v3-api store list``
+
+    Notice the output contains two Expectation stores: the original ``expectations_store`` on the local filesystem and the ``expectations_postgres_store`` we just configured.  This is ok, since Great Expectations will look for Expectations in PostgreSQL as long as we set the ``expectations_store_name`` variable to ``expectations_postgres_store``, which we did in the previous step.  The config for ``expectations_store`` can be removed if you would like.
+
+    ```bash
+    great_expectations --v3-api store list
+
+    - name: expectations_store
+    class_name: ExpectationsStore
+    store_backend:
+        class_name: TupleFilesystemStoreBackend
+        base_directory: expectations/
+
+    - name: expectations_postgres_store
+    class_name: ExpectationsStore
+    store_backend:
+        class_name: DatabaseStoreBackend
+        credentials:
+            database: '<your_db_name>'
+            drivername: postgresql
+            host: '<your_host_name>'
+            password: ******
+            port: '<your_port>'
+            username: '<your_username>'
+    ```
+
+
+5. **Create a new Expectation Suite by running** ``great_expectations --v3-api suite new``
+
+    This command prompts you to create and name a new Expectation Suite and to select a sample batch of data for the Suite to describe. Behind the scenes, Great Expectations will create a new table in your database called ``ge_expectations_store``, and populate the fields ``expectation_suite_name`` and ``value`` with information from the newly created Expectation Suite.
+
+    If you follow the prompts and create an Expectation Suite called ``exp1``, you can expect to see output similar to the following :
+
+    ```bash
+    great_expectations --v3-api suite new
+
+    #  ...
+
+    Name the new Expectation Suite: exp1
+
+    Great Expectations will choose a couple of columns and generate expectations about them
+    to demonstrate some examples of assertions you can make about your data.
+
+    Great Expectations will store these expectations in a new Expectation Suite 'exp1' here:
+
+    postgresql://'<your_db_name>'/exp1
+
+    #  ...
+    ```
+
+
+6. **Confirm that Expectations can be accessed from PostgreSQL by running** ``great_expectations --v3-api suite list``
+
+    The output should include the Expectation Suite we created in the previous step: ``exp1``.
+
+    ```bash
+    great_expectations --v3-api suite list
+
+    1 Expectation Suites found:
+     - exp1
+    ```
+
+
+If it would be useful to you, please comment with a +1 and feel free to add any suggestions or questions below.  Also, please reach out to us on [Slack](https://greatexpectations.io/slack) if you would like to learn more, or have any questions.
```

Comment:
@cdkini In my opinion, the title would read better with the following small change:
```How to configure an Expectation store to use PostgreSQL```
Thank you.

Reply:
I agree 💯 

I've gone ahead and updated the title, the filename, and any links from within the docs (`sidebar.js` and `customize-your-deployment.md`)